zerver/lib/gitter_import.py

```diff
@@ -210,8 +210,10 @@ def build_subscription(recipient_id: int, user_id: int,
                        subscription_id: int) -> ZerverFieldsT:
     subscription = dict(
         recipient=recipient_id,
-        notifications=False,
         color=random.choice(stream_colors),
+        audible_notifications=True,
+        push_notifications=False,
+        email_notifications=False,
         desktop_notifications=True,
         pin_to_top=False,
         in_home_view=True,
```

Comment:
@rheaparekh we should probably refactor the way subscription objects are created here to use the one we use for `UserProfile` where we're only setting a few fields with non-default values, rather than everything...